Project.toml

```diff
@@ -23,7 +23,9 @@ julia = "1"
 
 [extras]
 ChainRulesTestUtils = "cdddcdb0-9152-4a09-a978-84456f9df70a"
+JuMP = "4076af6c-e467-56ae-b986-b466b2749572"
+SCS = "c946c3f1-0d1f-5ce8-9dea-7daa1f7e2d13"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
```

Comment:
Dont we need Random here?

Reply:
I saw that Random was already in the package dependencies, but I don't think it's used outside of tests. I can move it to `extras`, unless there's something I'm missing.

Reply:
I'll do that in a following PR, we can leave it in the deps for now